src/components/preview/preview.pw.tsx

```diff
@@ -58,39 +59,36 @@ test.describe("check Preview component properties", () => {
     });
   });
 
-  test("should check when loading is True for Preview component", async ({
-    mount,
-    page,
-  }) => {
-    await mount(<PreviewComponent loading />);
-
-    await expect(previewComponent(page)).toBeVisible();
-  });
-
-  test("should check when loading is False for Preview component", async ({
-    mount,
-    page,
-  }) => {
-    await mount(<PreviewComponent loading={false} />);
+  [true, false].forEach((bool) => {
+    test(`should check when loading is set as ${bool} for Preview component`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<PreviewComponent loading={bool} />);
 
-    await expect(previewComponent(page)).not.toBeVisible();
+      if (bool === true) {
```

Comment:
**suggestion** (non-blocking): you could just do `if (bool)` instead of `if(bool === true)`, as it's slightly shorter and cleaner